ogre2/src/Ogre2RayQuery.cc

```diff
@@ -244,60 +339,188 @@ RayQueryResult Ogre2RayQuery::ClosestPointByIntersection(bool _forceSceneUpdate)
         meshName = meshName.substr(0, idx);
 
       const common::Mesh *mesh =
-           common::MeshManager::Instance()->MeshByName(meshName);
+        common::MeshManager::Instance()->MeshByName(meshName);
 
       if (!mesh)
         continue;
 
       Ogre::Matrix4 transform = ogreItem->_getParentNodeFullTransform();
 
+      const bool bIsAffine = transform.isAffine();
+
+      Ogre::Ray mouseRay;
+#ifndef SLOW_METHOD
```

Comment:
is this a new macro or something defined in ogre? Maybe add a brief comment / doc about this macro?

Reply:
same question goes for the `SINGLE_THREADED` macro

Reply:
I added explanations on the top.

These macros are basically "just in case" but (knocks on wood) should never be needed other than for benchmarking